neurom/ezy/neuron.py

```diff
@@ -111,6 +111,7 @@ def __init__(self, filename, iterable_type=np.array):
         self._nrn = load_neuron(filename, set_tree_type)
         self.soma = self._nrn.soma
         self.neurites = self._nrn.neurites
+        self.filename = self._nrn.id
```

Comment:
I think this should be called something like "label" or "name". Also, don't we want to strip it? It could get quite long if the filename path is long.